model/Makefile

```diff
@@ -30,21 +30,31 @@ test:
 docs/%.md: docs/%.qmd
 	poetry run quarto render $<
 
+# Pattern rule to build a reStructuredText file from a qmd file
+docs/%.rst: docs/%.qmd
+	poetry run quarto render $< --to rst
+	sed -i.bak 's/.. code:: mermaid/.. mermaid::/g' docs/$*.rst
```

Comment:
Where else is this done? Are you trying to replace [this](https://github.com/CDCgov/multisignal-epi-inference/blob/7822c37028776a2f609916fd13ca8c7106fb10d0/.github/workflows/website.yaml#L41-L50)?

Reply:
Not replace, but replicate.

Reply:
I see this has a thumbs up @dylanhmorris and @gvegayon. I would accept an approving review (possibly with a new issue to unify the RST post-processing) or a request for changes.

Reply:
I will review; can you create the issue about unified post-processing?